src/chainparams.cpp

```diff
@@ -400,10 +400,10 @@ class CTestNetParams : public CChainParams {
         nPruneAfterHeight = 1000;
         bnProofOfWorkLimit = arith_uint256(~arith_uint256() >> 16);
     
-        uint32_t nTimestamp = 1545341312;
-        uint256 hashGenesisBlock = uint256S("0x0000a2ed763c6efc24bbb3ac8d9f1ab9e8f1e7100d5221ad80815cd7b369dc2c");
-        uint256 hashMerkleRoot = uint256S("0x02128838f2516796eb04f5b3fd143a7786001301dc5ffcfd2b2c687a2864aae9");
-        uint32_t nNonce = 2043585747;
+        uint32_t nTimestamp = 1573858751;
```

Comment:
IDK why this diff is here when PR #628 as been merged?

Reply:
@proletesseract somehow the diff did not update, I merged master into this PR and it seems to have fixed the diff on github.